cmd/spec/example_reservation.go

```diff
@@ -86,7 +86,7 @@ var AwsReservationRequestPayloadExample = payloads.AWSReservationRequest{
 }
 
 var AwsReservationResponsePayloadPendingExample = payloads.AWSReservationResponse{
-	PubkeyID:         42,
+	PubkeyID:         func() *int64 { i := int64(42); return &i }(),
```

Comment:
This is interesting, please explain why you haven’t used our utility function `ptr.ToInt64(42)` or alternatively `ptr.To(int64(v))`.

Reply:
Oh yeah! I definitely overlooked the ptr.ToInt64 function. This makes perfect sense to use since we have it. Thanks for the comment!